src/test/java/de/tracetronic/jenkins/plugins/ecutest/extension/jobdsl/AbstractDslExtensionIT.java

```diff
@@ -47,28 +49,25 @@ protected FreeStyleProject createTestJob() throws Exception {
         buildSeedJob(getDslScript());
 
         assertThat(jenkins.getInstance().getJobNames(), hasItem(is(getJobName())));
-        final FreeStyleProject project = jenkins.getInstance().getItemByFullName(getJobName(), FreeStyleProject.class);
-        return project;
+        return jenkins.getInstance().getItemByFullName(getJobName(), FreeStyleProject.class);
```

Comment:
jenkins.get().getItemByFullName(getJobName(), FreeStyleProject.class);

Reply:
`jenkins` derives from `JenkinsRule` here and there is no `get()` method.